screens/rewards/RewardsScreen.js

```diff
@@ -144,9 +144,9 @@ export default class RewardsScreen extends React.Component {
     return (
       <Container>
         <TopTab>
-          <BackButton onPress={() => this.props.navigation.goBack()}>
+          <NavButton onPress={() => this.props.navigation.goBack()}>
             <FontAwesome5 name="arrow-down" solid size={24} color="white" />
-          </BackButton>
+          </NavButton>
```

Comment:
Button isn't center-aligned / seems to be flushed to the bottom of the header, and looks like it is sticking right above the words "Healthy Rewards".
![image](https://user-images.githubusercontent.com/35573990/77873156-b74c5b80-727b-11ea-9807-817c9b50cb3b.png)

Reply:
Interesting! It is actually properly aligned but does not appear to be because the icon has a 40px width and some padding, making the icon itself only 22px centered in the 40px so it appears misaligned. But thank you for noticing this! I will adjust it in #56